hack/api-reference/calico.md

```diff
@@ -67,7 +67,7 @@ IPAM
 </td>
 <td>
 <em>(Optional)</em>
-<p>IPAM to use for the Calico Plugin (e.g., host-local or Calico)</p>
+<p>IPAM to use for the Calico Plugin (e.g., host-local or calico-ipam)</p>
```

Comment:
Please do not update the documentation directly. This is generated from the type documentation.

Therefore, please update `pkg/apis/calico/v1alpha1/types_network.go` and `pkg/apis/calico/types_network.go` instead and run `make generate` afterwards.

Reply:
@ScheererJ done

Reply:
Not really. Only one of the two files was changed.

Reply:
@ScheererJ third time is the charm! 😅